File: pset6/cash.py
from cs50 import get_float
while True:
    dollars = get_float("Change owned: ")
    if dollars > 0:
        break

cent = round(dollars*100)
quarter = cent // 25
dime = (cent -(quarter * 25)) // 10
nickel = (cent -(quarter * 25) - (dime * 10)) // 5
penny = cent - (quarter * 25) - (dime * 10) - (nickel * 5)
coins = quarter + dime + nickel + penny
print(coins)

File: pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void){
    float dollars;
    do{
    dollars=get_float("how much changes is owed($):\n");
    }while(dollars<=0.0);

    int cent = round(dollars*100);
    int quarter = cent / 25;
    int dime = (cent -(quarter * 25)) / 10;
    int nickel = (cent -(quarter * 25) - (dime * 10)) / 5;
    int penny = cent - (quarter * 25) - (dime * 10) - (nickel * 5);
    int coins = quarter + dime + nickel + penny;
    printf("%i\n", coins);
   

}
